ntp-proto/src/peer.rs

```diff
@@ -115,10 +115,9 @@ pub struct PeerSnapshot {
     pub(crate) reference_timestamp: NtpTimestamp,
     #[allow(dead_code)]
     pub(crate) poll_interval: PollInterval,
```

Comment:
these `dead_code` fields are not needed after all?

Reply:
probably a good idea, I will make a PR for that

Reply:
Removed by #92